@claude CI fails with:

```
error[E0405]: cannot find trait `UniffiCustomTypeConverter` in crate `uniffi`
  --> dash-spv/src/bridge/mod.rs:16:14
```

The `UniffiCustomTypeConverter` trait doesn't exist in UniFFI 0.28. In this version, the simple two-argument form `uniffi::custom_type!(Network, String)` should work on its own — it auto-generates conversions using `From`/`Into` traits. Since `dashcore::Network` implements `Display` and `FromStr`, just use:

```rust
use dashcore::Network;
uniffi::custom_type!(Network, String);
```

Remove the `impl UniffiCustomTypeConverter` block entirely and just keep the simple macro call. If `From<Network> for String` and `TryFrom<String> for Network` aren't implemented, you may need to add them. Push the fix.

--- a/dash-spv/Cargo.toml
+++ b/dash-spv/Cargo.toml
@@ -56,7 +56,7 @@ tempfile = { version = "3.0", optional = true }
 dashcore-rpc = { path = "../rpc-client", optional = true }
 
 # UniFFI (optional, for mobile bridge validation)
-uniffi = { version = "0.28", optional = true }
+uniffi = { version = "0.31.0", optional = true }
 
 # DNS (trust-dns-resolver was renamed to hickory_resolver)
 hickory-resolver = "0.25"

--- a/dash-spv/src/bridge/mod.rs
+++ b/dash-spv/src/bridge/mod.rs
@@ -14,7 +14,7 @@ use dashcore::Network;
 uniffi::custom_type!(Network, String, {
     remote,
     lower: |n| n.to_string(),
-    try_lift: |s| s.parse().map_err(Into::into),
+    try_lift: |s| s.parse().map_err(|e| uniffi::deps::anyhow::anyhow!(e)),
 });
 
 /// A simple sync function that returns a greeting string.

--- a/dash-spv/src/bridge/mod.rs
+++ b/dash-spv/src/bridge/mod.rs
@@ -11,7 +11,11 @@ use std::sync::Arc;
 
 use dashcore::Network;
 
-uniffi::custom_type!(Network, String);
+uniffi::custom_type!(Network, String, {
+    remote,
+    lower: |n| n.to_string(),
+    try_lift: |s| s.parse().map_err(Into::into),
+});
 
 /// A simple sync function that returns a greeting string.
 #[uniffi::export]

--- a/dash-spv/src/bridge/mod.rs
+++ b/dash-spv/src/bridge/mod.rs
@@ -13,18 +13,6 @@ use dashcore::Network;
 
 uniffi::custom_type!(Network, String);
 
-impl uniffi::UniffiCustomTypeConverter for Network {
-    type Builtin = String;
-
-    fn into_custom(val: Self::Builtin) -> uniffi::Result<Self> {
-        val.parse().map_err(|e| anyhow::anyhow!("Invalid network: {e}").into())
-    }
-
-    fn from_custom(obj: Self) -> Self::Builtin {
-        obj.to_string()
-    }
-}
-
 /// A simple sync function that returns a greeting string.
 #[uniffi::export]
 pub fn hello() -> String {

